content/tutorial/tutorial.md

```diff
@@ -12,185 +12,190 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+यह ट्यूटोरियल किसी भी मौजूदा React ज्ञान को ग्रहण नहीं करेगा।
```

Comment:
ग्रहण would not be the correct translation of assumption

Reply:
how about मौजूदा ज्ञान की कोई आवश्यकता नहीं है?

Reply:
As per english docs, this will be more closer meaning.
`इस टुटोरिअल के लिए React के मौजूदा ज्ञान की आवश्यकता नहीं है`

Reply:
"any" existing React knowledge better maps to कोई आवश्यकता नहीं है

Reply:
But we should also include `React` word in the sentence. As that will make more sense

Reply:
इस टुटोरिअल के लिए React के मौजूदा ज्ञान की कोई आवश्यकता नहीं है is what I meant

Reply:
Oh. ok. This makes more sense. Let me update this